src/components/TimezoneSelect.vue

```diff
@@ -13,27 +13,15 @@
   import { titleCase } from '@/utilities/strings'
   import { formatDateInTimezone, utcTimezone } from '@/utilities/timezone'
 
+  const modelValue = defineModel<string | null>({ required: true })
+
   const props = defineProps<{
-    modelValue: string | null,
     showTimestamp?: boolean,
     hideUnset?: boolean,
   }>()
 
-  const emit = defineEmits<{
-    (event: 'update:modelValue', value: string | null): void,
-  }>()
-
-  const internalValue = computed({
-    get() {
-      return props.modelValue ?? 'UTC'
```

Comment:
Specifically removing the `?? 'UTC'` is the fix here. Updating the syntax to use `defineModel` as well.